build.sbt

```diff
@@ -18,13 +18,14 @@ lazy val microservice = (project in file("."))
   .settings(PlayKeys.playDefaultPort := 9581)
```

Comment:
This file probably needs a cleanup at some point. However, can you please add `.settings(integrationTestSettings(): _*)`after[ here](https://github.com/hmrc/tariff-classification-frontend/blob/main/build.sbt#L68) to generate headers for integration test files. It might have been missed because the scalastyle config file needs updating. Thanks

Reply:
Also please remove the resolver [here](https://github.com/hmrc/tariff-classification-frontend/blob/main/build.sbt#L80-L82). Thanks